index.md

```diff
@@ -65,6 +65,18 @@ It looks like you are setting up a website for a Data Carpentry curriculum but y
 {% endunless %}
 {% endif %}
 
+{% comment %}
+Check SWC curriculum
+{% endcomment %}
+
+{% if site.carpentry == "swc" %}
+{% unless site.curriculum == "swc-inflammation" or site.curriculum == "swc-gapminder" %}
+<div class="alert alert-warning">
+It looks like you are setting up a website for a Software Carpentry curriculum but you haven't specified the curriculum type in the <code>_config.yml</code> file (current value in <code>_config.yml</code>: "<strong>{{ site.curriculum }}</strong>", possible values: <code>swc-inflammation</code>, or <code>swc-gapminder</code>). After editing this file, you need to run <code>make serve</code> again to see the changes reflected.
+</div>
+{% endunless %}
+{% endif %}
+
```

Comment:
After this change, we will see the above warning on this website.

Reply:
For what it's worth, we currently already see [an alert on the site](https://github.com/carpentries/workshop-template/pull/664/commits/592b0d0af149bc7d2c214893ebc67532ff2fdea6#diff-d680e8a854a7cbad6d490c445cba2ebaL42-L48), so another alert below it will not be too detrimental to the current status quo of the site.

The question becomes: who is the audience for https://carpentries.github.io/workshop-template? I know at the moment, several lessons point to the install page for updated installation instructions, but am I wrong in thinking that learners would not be visiting this site otherwise? Is there a better way to fail fast, early, and obviously so that the facilitators can set up the website properly?

Reply:
We can hide divs that belong to class alert and alert-danger using jQuery:
```js
<script src="https://code.jquery.com/jquery-3.5.0.js"></script>
<script>
$( document ).ready(function() {
  if (location.href.startsWith("https://carpentries.github.io/workshop-template/")) {
      $("div.alert.alert-danger").hide();
  }
});
</script>
```

Reply:
> We can hide divs that belong to class alert and alert-danger using jQuery

If we didn't want the alerts to appear on the page, we could just not place them there in the first place.

To determine if we should have alerts like this on the page, we have to ask: who is the target audience and how does it help us to have the alerts?

Reply:
The alerts have to appear on the workshop (event) page if preparer forgot to update the fields but they don't have to appear on the workshop-template's website.